spec/Vehicle/Vehicle.vspec

```diff
@@ -370,15 +370,15 @@ CurrentLocation.Latitude:
   min: -90
   max: 90
   unit: degrees
-  description: Current latitude of vehicle according to WGS 84.
+  description: Current latitude of vehicle according to WGS 84, as measured at the position of GNSS receiver antenna.
```

Comment:
> according to WGS 84

maybe better: "in WGS84 geodetic coordinates"

Reply:
Works fine for me, but it seems that they use `WGS 84` (with space) in the standard rather than `WGS84` (without space) so I will keep the space